src/nix/install.sh

```diff
@@ -12,12 +12,6 @@ FLAKEURI="${FLAKEURI:-""}"
 EXTRANIXCONFIG="${EXTRANIXCONFIG:-""}"
 USERNAME="${USERNAME:-"${_REMOTE_USER:-"automatic"}"}"
 
-# Nix keys for securely verifying installer download signature per https://nixos.org/download.html#nix-verify-installation
```

Comment:
Can we bump the patch version? 

https://github.com/devcontainers/features/blob/3921f5917148583af38d1d96a1b7e0f31d22181c/src/nix/devcontainer-feature.json#L3

Reply:
resolved in aa00f714313489f3d3eb2c3c738870023cf6302d